[WIP] Follow #25 and more progress (#26)
v-community/v_by_example: examples/functions/functions.md

}
```

> ## Variadic Functions

Reviewer: 👍

Functions can also be variadic i.e. accept infinite number of arguments. They are **not** arrays and cannot be returned.

```go
fn main() {

> 

Reviewer: Extra row. I'd consider removing it.

}
```

## Variadic Functions

Functions can also be variadic i.e. accept infinite number of arguments. They are **not** arrays and cannot be returned.

```go
fn main() {
	foo("V", "is", "the", "best", "lang" , "ever")	
}

fn foo(test ...string) {
	println(test)
}
```

Output

```bash
V
is
the
best
lang
ever
```

## Multi-Return Functions

> Similar to Go, functions in V can also return more than one and with different type.

Reviewer: ```suggestion
Similar to Go, functions in V can also return multiples values with different data types.
```
